service/routes.py

```diff
@@ -59,159 +59,200 @@ def index():
 
 # query string arguments
 inventory_args = reqparse.RequestParser()
+inventory_args.add_argument('name', type=str, location='args', required=False, help='List Inventory Items by Name')
+inventory_args.add_argument('condition', type=str, location='args', required=False, help='List Inventory Items by Condition')
+inventory_args.add_argument('restock', type=int, location='args', required=False, help='List Inventory Items by Restock Level')
```

Comment:
```suggestion
inventory_args.add_argument('restock', type=inputs.boolean, location='args', required=False, help='List Inventory Items by Necessity to Restock')
```

Reply:
This needs to be fixed as a bug, because the model takes a string as an input. Will have to fix this in the next PR